frontend/src/metabase/parameters/components/ParameterLinkedFilters/use-filter-fields.ts

```diff
@@ -20,7 +21,7 @@ const useFilterFields = (
     const filtered = getParameterFieldIds(parameter);
     const filtering = getParameterFieldIds(otherParameter);
 
-    if (!filtered.length || !filtering.length) {
+    if (!filtered.length || !filtered.length) {
```

Comment:
🤔 This is the same condition though. Not sure if this got changed during the refactoring or not, but this seems odd.

I'd expect this to either be just:
```suggestion
    if (!filtered.length) {
```

or leave it as is
```suggestion
    if (!filtered.length || !filtering.length) {
```